Wikipedia/Code/EventLoggingFunnel.m

```diff
@@ -28,8 +29,7 @@ - (NSDictionary*)preprocessData:(NSDictionary*)eventData {
 }
 
 - (void)log:(NSDictionary*)eventData {
-    SessionSingleton* session = [SessionSingleton sharedInstance];
-    NSString* wiki            = [session.currentArticleSite.language stringByAppendingString:@"wiki"];
```

Comment:
good catch - i thought i avoided all the "article specific" ones… but this one might be… I'll back it out here.